jenkins/Jenkinsfile

```diff
@@ -31,6 +31,30 @@ pipeline {
             '''
           }
         }
+        stage('NDTensorsCUDAExt julia-1') {
+	        options {
+            timeout(time: 45, unit: 'MINUTES')
+          }
+          agent {
+            dockerfile {
+              label 'gpu&&v100'
+              filename 'Dockerfile'
+              dir 'jenkins'
+              additionalBuildArgs  '--build-arg JULIA=1'
+              args '--gpus "device=1"'
+            }
+          }
+          environment {
+            HOME = pwd(tmp:true)
+            OMP_NUM_THREADS = 4
+            JULIA_NUM_THREADS = 4
+          }
+          steps {
+            sh '''
+              julia -e 'using Pkg; Pkg.activate(temp=true); Pkg.Registry.update(); Pkg.update(); Pkg.develop(path="./NDTensors"); Pkg.develop(path="."); Pkg.test("NDTensors"; test_args=["cuda"])'
+            '''
+          }
+        }
         stage('NDTensorscuTENSORExt julia-1.10') {
```

Comment:
```suggestion
        stage('NDTensorscuTENSORExt julia-lts') {
```

Reply:
Or maybe not here since this workflow is hard coded to Julia 1.10. Ideally we would switch to `juliaup` which supports the keyword `lts`.

Reply:
I think we would need to ask Dylan to install juliaup on the docker for the linux builds. I'll try lts and see if it fails

Reply:
It looks the code right now tries to just stick in `JULIA=#` into the .gz path. for example
`https://julialang-s3.julialang.org/bin/linux/x64/lts/julia-lts-latest-linux-x86_64.tar.gz `

Reply:
There is no `lts` version in a tar ball so I reverted to 1.10

Reply:
Thanks for checking, seems like this is the best we can do right now without something more involved. Yeah, we should contact Dylan about it if it doesn't work right now, `juliaup` would be a better way to go in all of the workflows.